docs/UserGuide.md

```diff
@@ -43,26 +43,37 @@ InsureBook is a **desktop app for insurance agents to manage potential and exist
 ## Quick start
 
 
-1. Ensure you have Java `11` or above installed in your Computer.
-   - If your computer is not running Java version `11` or above, you may download it from [here](https://www.oracle.com/java/technologies/downloads/#java11).
+1. Ensure you have Java `11` or above installed in your Computer.<br>
+   If your computer is not running Java version `11` or above, you may download it from [here](https://www.oracle.com/java/technologies/downloads/#java11).
 2. Download the latest `InsureBook.jar` from [here](https://github.com/AY2324S2-CS2103T-W09-4/tp/releases).
 3. Copy the file to the folder you want to use as the _home folder_ for InsureBook.
 4. Open a command terminal and `cd` into the folder you put the jar file in. Alternatively, <br>
-   **[Windows]** right-click anywhere in the display pane's background, then click "Open in Terminal":
+   **[Windows 11]** right-click anywhere in the display pane's background, then click "Open in Terminal":
```

Comment:
The button is named differently for different versions. I'll add a line to mention our assumption that **users are proficient in CLI-based interactions**, since that is our target user, and remove these screenshots to be safe.